Arrowgene.Ddon.Shared/Files/Assets/Shop.json

```diff
@@ -4928,7 +4928,7 @@
             "GoodsParamList": [
                 {
                     "Index": 0,
-                    "ItemId": 1074,
+                    "ItemId": 21825,
```

Comment:
Is this correct or was this for testing and snuck it's way into the commit 😛

Reply:
It is not correct, but it is also still a draft :).